buildSrc/src/main/java/org/elasticsearch/test/NamingConventionsCheck.java

```diff
@@ -47,10 +44,12 @@
  */
 public class NamingConventionsCheck {
     public static void main(String[] args) throws IOException, ClassNotFoundException {
-        NamingConventionsCheck check = new NamingConventionsCheck();
+        int i = 0;
+        NamingConventionsCheck check = new NamingConventionsCheck(
+                Thread.currentThread().getContextClassLoader().loadClass(args[i++]),
```

Comment:
The rest of the code seems to do it this way, but, yeah, I suspect it doesn't need to do it that way.